src/core/gltf2importer/meshparser.cpp

```diff
@@ -123,6 +125,201 @@ Qt3DRender::QAttribute *decodeAttribute(const draco::PointCloud *pointCould,
     return attribute;
 }
 #endif
+
+int vertexBaseTypeSize(Qt3DRender::QAttribute::VertexBaseType vertexBaseType)
+{
+    switch (vertexBaseType) {
+    case Qt3DRender::QAttribute::UnsignedByte:
+        return 1;
+    case Qt3DRender::QAttribute::UnsignedShort:
+        return 2;
+    case Qt3DRender::QAttribute::UnsignedInt:
+        return 4;
+    }
+}
+
+
+std::array<unsigned int, 3> findVertexIndicesInFace(const Qt3DRender::QAttribute *indexAttr, const int iFace) {
+    const auto byteOffset = indexAttr->byteOffset();
+    const auto byteStride = indexAttr->byteStride() == 0 ?
+                            3*vertexBaseTypeSize(indexAttr->vertexBaseType()) :
```

Comment:
Spaces around *

There are more of those, but I'll ignore style issues in the rest of this review. Really, get the clang-format integration with git commit or your IDE so that it does it for you.